environ.py

```diff
@@ -0,0 +1,29 @@
+# standard libraries
+import json, logging, os
+
+# entry-level job modules need to be one-level beneath root
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+logger = logging.getLogger(__name__)
+
+
+# Set up ENV
+
+config_path = os.path.join(ROOT_DIR, os.getenv('ENV_PATH', os.path.join('config', 'secrets', 'env.json')))
+
+try:
+    with open(config_path) as env_file:
+        ENV = json.loads(env_file.read())
+except FileNotFoundError:
+    logger.error('Configuration file could not be found; please add env.json to the config directory.')
+    ENV = None
+
+logging.basicConfig(level=ENV['LOG_LEVEL'])
+
+# Add ENV key-value pairs to environment, skipping if the key is already set
+logger.debug(os.environ)
+for key, value in ENV.items():
+    if key in os.environ:
+        ENV[key] = value
+        logger.info('ENV value overidden')
+        logger.info(f'key: {key}; value: {value}')
```

Comment:
This can't handle collection objects. Do you think I should have a JSON load step in here?

Reply:
Wait, also there's a bug, blah

Reply:
Okay, this should be fixed now. I'll do some testing.

Reply:
Hmm, do you mean configure environment variables as JSON and load them? We could [try/except with a json.loads](https://www.geeksforgeeks.org/python-check-whether-a-string-is-valid-json-or-not/) and if it excepts just set the ENV[key] = value otherwise set it to the json value? 

Like if we had an environment variable.
`UDW = {"host": "", "port": "","dbname": "","user": "", "password": ""}`

This would load up the json, but if we had 
CANVAS_TERM_ID = 164 that would also work.

Reply:
Just did that, as well as fixed the bug I mentioned.
See commit https://github.com/tl-its-umich-edu/course-inventory/pull/62/commits/eadda6888773eb69085425b02e47e6a6a9088a79